capture.py

```diff
@@ -688,18 +688,37 @@ def _applied_viewport_options(options, panel):
     # be set differently (see #55)
     plugins = cmds.pluginDisplayFilter(query=True, listFilters=True)
     plugin_options = dict()
+    old_plugin_options = dict()
     for plugin in plugins:
         if plugin in options:
             plugin_options[plugin] = options.pop(plugin)
-    
-    # default options
+            old_plugin_options[plugin] = cmds.modelEditor(
+                panel, query=True, queryPluginObjects=plugin)
+
+    # get old options
+    old_options = dict()
+    for opt in options.copy():
```

Comment:
You probably copied this from elsewhere in the code - but in this particular case you don't need to iterate over a copy of `options` because you're not changing it (like popping) in-place. In this case you're only editing another dictionary.